src/SIL.LCModel.Build.Tasks/SIL.LCModel.Build.Tasks.csproj

```diff
@@ -32,11 +32,11 @@ SIL.LCModel.Build.Tasks provides msbuild tasks for generating C# classes for the
     <ItemGroup>
       <None Include="$(PackageId).props" Pack="true" PackagePath="build/" />
       <None Include="$(PackageId).props" Pack="true" PackagePath="buildMultiTargeting/" />
-      <None Include="$(OutputPath)/net461/antlr.runtime.dll" Pack="true" PackagePath="tools/net461/" />
-      <None Include="$(OutputPath)/net461/IDLImporter.dll" Pack="true" PackagePath="tools/net461/" />
-      <None Include="$(OutputPath)/net461/IDLImporter.xml" Pack="true" PackagePath="tools/net461/" />
-      <None Include="$(OutputPath)/net461/Newtonsoft.Json.dll" Pack="true" PackagePath="tools/net461/" />
-      <None Include="$(OutputPath)/net461/NVelocity.dll" Pack="true" PackagePath="tools/net461/" />
+      <None Include="$(OutputPath)/antlr.runtime.dll" Pack="true" PackagePath="tools/" />
```

Comment:
IMO it would be better to put the files in `tools/netstandard2.0` (or whatever the correct directory name is) in the nuget package. That way it's easier to see that the files got build for netstandard2.0.

Reply:
If I'm understanding you, that would just be changing the PackagePath for these third-party lib's back to including the target framework. This should be simple, but at first glance I wonder: do you think it will lead people to believe that those lib's were built for netstandard2.0? They may not be.

Reply:
> do you think it will lead people to believe that those lib's were built for netstandard2.0? They may not be.

Good point. The main dll is build for netstandard2.0 though. But that raises another interesting question: if the other libs are not build for netstandard2.0, will they work in all environments that can consume netstandard2.0? I'm thinking especially about .NET Core 3.1.

But that might not be terribly important since this dll defines msbuild tasks anyway which are then used in a process running `msbuild` or `dotnet build`. And we can probably require that the machine building that client project has a newer framework version installed, so I assume we can ignore this problem.

Reply:
In short, I don't know. Per [this link](https://stackoverflow.com/a/46394435), it is quite possible, but it depends totally on what calls are being made in the Framework lib's and whether Core 3.1 API supports/provides those calls or not.